model_hub/examples/mmdetection/yolov3.yaml

```diff
@@ -45,6 +45,6 @@ environment:
     - OMP_NUM_THREADS=1 # Following pytorch dtrain, this environment variable is set to 1 to avoid overloading the system.
 
 resources:
-  slots_per_trial: 8 # max number of GPUs a trial is allowed to individually use
+  slots_per_trial: 2 # max number of GPUs a trial is allowed to individually use
```

Comment:
don't forget to restore this before landing this pr

Reply:
added e2e tests for multi-node torch experiments